src/node/interfaces.cpp

```diff
@@ -971,24 +971,32 @@ class MinerImpl : public Mining
         return BlockRef{tip->GetBlockHash(), tip->nHeight};
```

Comment:
In commit message of ad3af401c19b8d05ce69011a359db3090b7018e1 `rpc: handle shutdown during long poll and wait methods`:

```diff
- getblocktemplate,  waitfornewblock
+ getblocktemplate, waitfornewblock
```